tests/test_timeouts.py

```diff
@@ -21,6 +24,11 @@
 
 
 async def test_write_timeout(server, backend):
+    # FIX: win32 doesn't raise asyncio.TimeoutError
+    # in 'asyncio.wait_for(writer.drain(), timeout.write_timeout)'
+    if sys.platform == "win32":
+        if isinstance(backend, AsyncioBackend):
+            pytest.skip()
```

Comment:
If this test is expected to fail, would `xfail` be better suited here?

We can also add the comment content into the `reason` so that it is reported at the end of the test session by pytest.

```suggestion
            pytest.xfail(
                "'asyncio.wait_for(writer.drain())' does not raise a 'TimeoutError' on Windows."
            )
```